eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -30,6 +30,12 @@ parameters:
 - name: TestSetupSteps
   type: stepList
   default: []
+- name: WindowsPool
+  type: string
+  default: azsdk-pool-mms-win-2019-general
```

Comment:
Is there a clean way to toggle between a machine pool or an os image at the top level?

Reply:
We don't need to (currently) specify the vmImage at the top level. We special case macOS deeper down because we don't have a non Azure DevOps hosted pool option there. But for Windows and Linux we should be using the 1ES pools moving forward which is presently 1:1 image to pool mapping.

Even when/if they support multiple images per pool I'm not 100% certain that we'll take advantage of it since it probably doesn't give us any benefits over just having multiple pools (in fact I think pool per image would end up providing better throughput underload).